bundle/edu.gemini.p2checker/src/test/scala/edu/gemini/p2checker/rules/GmosSDtaXRuleTest.scala

```diff
@@ -58,9 +58,16 @@ class GmosSDtaXRuleTest extends AbstractRuleTest {
 
   private def expectSuccess(): Unit = {
 //    println(runCheck.mkString(",\n"))
-    assertTrue(runCheck.isEmpty)
+    assertTrue(! runCheck.exists { p =>
```

Comment:
Small thing, from a style point of view perhaps we prefer no space on the negation `!runCheck.ex...`